test-data/unit/stubgen.test

```diff
@@ -355,12 +396,17 @@ __all__ = ['p']
 import urllib.parse as p
 
 [case testExportModuleInPackageUnsupported_import]
-import urllib.parse
 __all__ = ['urllib']
 [out]
 # Names in __all__ with no definition:
 #   urllib
 
+[case testExportModuleInPackage_import]
+import urllib.parse
+__all__ = ['urllib']
+[out]
+import urllib.parse
```

Comment:
Running `import urllib.parse` defines the `urllib` name and attaches it to the correct module, why this should be an error?